shared/openId4VCI/Utils.ts

```diff
@@ -23,6 +23,17 @@ export const Issuers = {
   ESignet: 'ESignet',
 };
 
+/**
+ * @param issuer the issuer of the VC as per the VC metadata in MMKV
+ * @returns the ID-type to be used for further translation
+ */
+export function getIdType(issuer: string | undefined): string {
```

Comment:
why do we need to explicitly mention NATIONAL_ID and INSURANCE_ID, can't it be handled using 
ID_TYPE

Reply:
This is a temporary change, I should document it likewise.
Right now I'm using this field to figure out the kind of card it is.

Reply:
Got it, you want me to use this?

```typescript
export const ID_TYPE = {
  MOSIPVerifiableCredential: i18n.t('VcDetails:nationalCard'),
  InsuranceCredential: i18n.t('VcDetails:insuranceCard'),
};
```

Reply:
I've implemented this.